lib/data/data_source/remote/authentication_remote_source_impl.dart

```diff
@@ -27,17 +27,7 @@ class AuthenticationRemoteSourceImpl implements AuthenticationRemoteSource {
         password: dto.password,
       );
       if (userId != null) {
-        final dto = UserProfileDto(
-          name: '',
-          userId: userId!,
-          initialLanguage: 'pl',
-          userFolders: [],
-          email: firebaseAuth.currentUser?.email ?? '',
-          appLanguage: 'pl',
-          nativeLanguage: 'pl',
-          languageToLearn: 'en',
-        );
-        await firestore.collection("users").doc(userId).set(dto.toJson());
+        _createUserDoc(user);
```

Comment:
this is not async function?

Reply:
The function '_createUserDoc' is a asynchron function